chapters/11-version-control-workflow.Rmd

```diff
@@ -1,18 +1,244 @@
-# Version control: Workflows around using GitHub (part 2) {#github-workflows}
+# Workflows when using GitHub {#github-workflows}
 
-- Make use of R builtin tools like the usethis pull request helpers
-  (and learning how to make use of this yourself)
-- E.g. with pr_* functions from usethis
-- Using this workflow as a team or as a contributor
+<!-- Brief summary of list of actions/code/concepts done -->
+
+- Exercises: Conceptual, how can people effectively collaborate on a project?
+    - Maybe get them to create a vignette article with their answers/thoughts? 
+    Articles that are ignored from package build. (we discussed this a bit
+    already, needs more discussion)
+- Branching/forks
+    - Text: what is it, power of it, features.
+- Pull Requests
+    - Exercises: Reading, comprehension, and conceptual understanding (non-coding)
+- Security:
+    - Text: why its important, how to do it.
+    - Code:
+        - usethis::git_sitrep()
+        - usethis::gh_token_help()
+        - usethis::create_github_token()
+    - Text: Save PAT in password manager (they should have this anyway, but this forces them)
+    - Code: 
+        - gitcreds::gitcreds_set(), everytime computer restarts, only saved for session
+    - Exercises: Conceptual understanding of PAT, tokens, and password managers (non-coding)
+    - Exercises: Setting up PAT, saving PAT, using gitcreds at start of session
+- usethis pr_* helpers
+    - Text: Why use these over GitHub interface
+- As contributor/sole developer/as team
```

Comment:
Q: Are all of this functions going to be demoed and explained, or just listed?

Reply:
They will be demoed and explained. Listing htem for now to show what will be used and the steps taken.